bundles/org.openhab.binding.linky/src/main/resources/OH-INF/i18n/linky_fr.properties

```diff
@@ -1,17 +1,19 @@
 # binding
 
 binding.linky.name = Extension Linky
-binding.linky.description = Cette extension collecte vos données de consommation d'énergie depuis le site internet d'Enedis.
+binding.linky.description = Cette extension collecte vos donn�es de consommation d'�nergie depuis le site internet d'Enedis.
```

Comment:
Please do not update the French translation in this PR.
By the way, you apparently even introduce a problem, maybe due to an invalid file encoding:
![image](https://user-images.githubusercontent.com/10186704/161382411-013085b2-7890-4a1f-8b69-3e4b71d7ca70.png)

Reply:
Oh, shit ! I corrected all the file thinking I introduced an incorrect encoding. I'll revert it to original version and we'll take care of it in crowdin.